pkg/spanconfig/spanconfigkvsubscriber/testdata/system_span_configs

```diff
@@ -0,0 +1,127 @@
+
+start
+----
+
+update
+upsert [a,c):A
+upsert [d,f):D
+----
+
+updates
+----
+/M{in-ax}
+[a,c)
+[d,f)
+
+store-reader key=a
+----
+conf=A
+
+store-reader key=e
+----
+conf=D
+
+store-reader key=g
+----
+conf=MISSING
+
+update
+upsert {entire-keyspace}:_CLUSTER
+----
+
+updates
+----
+/M{in-ax}
+
+store-reader key=a
+----
+conf=A_CLUSTER
+
+store-reader key=e
+----
+conf=D_CLUSTER
+
+# The system span config we set should still apply.
+store-reader key=g
+----
+conf=MISSING_CLUSTER
+
+# Update span configs that target tenant keyspaces and ensure handlers are
+# invoked correctly.
+update
+upsert {source=1, target=1}:_HOST
```

Comment:
[nit] Here too I'd prefer using single letters instead and showing combination of configs through "A+B+C".